ns/nns.go

```diff
@@ -116,50 +81,23 @@ func (n *NNS) Dial(address string) error {
 //
 // See also https://docs.neo.org/docs/en-us/reference/nns.html.
 func (n *NNS) ResolveContainerName(name string) (cid.ID, error) {
-	res, err := n.neoClient.call(n.nnsContract, "resolve", []smartcontract.Parameter{
-		{
-			Type:  smartcontract.StringType,
-			Value: name + ".container",
-		},
-		{
-			Type:  smartcontract.IntegerType,
-			Value: big.NewInt(int64(nns.TXT)),
-		},
-	})
+	arr, err := unwrap.Array(n.invoker.Call(n.nnsContract, "resolve",
```

Comment:
Do you still need to handle `Null`?

Reply:
did not have time to fix that before i faced that approve rain, added commit: https://github.com/nspcc-dev/neofs-sdk-go/pull/328/commits/5a279847959ace493ad9b3ca648b8f1d6b8ff272

/cc @alexvanin @KirillovDenis @fyrchik @masterSplinter01